packages/backend/src/kit/models/errors.ts

```diff
@@ -0,0 +1,6 @@
+export class KitAlreadyExistsError extends Error {
```

Comment:
Also here I suggest you to  implement HttpError object from `@map-colonies/error-express-handler`.

Reply:
same as above